content/php-client/introduction.md

```diff
@@ -6,7 +6,7 @@ Two clients exist:
 - a **Community Edition** client, covering the API endpoints of the PIM Community Edition
 - an **Enterprise Edition** client, covering the API endpoints of both the PIM Community Edition and the PIM Enterprise Edition
 
-The actual version of these clients is v1.0. It is compatible with **our dear PIM 2.0**.
+There are two versions of these clients. The version v1.0 is compatible with our PIM 2.0. The actual version v2.0 is compatible with **our dear PIM 2.1** and covers more endpoints than the previous version.
```

Comment:
Do you mean 'current' instead of 'actual' ?

Reply:
maybe "latest". Also, don't hesitage to put a carriage return.

The version v1.0 is compatible with our PIM 2.0.
The current and lastest version v2.0 is compatible with **our dear PIM 2.1** and covers more endpoints than the previous version.